<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setcl</class>
 <widget class="QDialog" name="setcl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>动作配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 设置对话框的背景色和边框 */
QDialog {
    background-color: #f4f4f4;
    border: 1px solid #b0b0b0;
}

/* 设置 QLabel 的字体和颜色 */
QLabel {
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #333333;
}

/* 设置 QRadioButton 的字体和颜色 */
QRadioButton {
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #555555;
}

/* 设置 QLineEdit 的背景色、边框和字体 */
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 5px;
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #333333;
}

/* 设置 QPushButton 的背景色、边框、字体和鼠标悬停效果 */
QPushButton {
    background-color: #0078d4;
    border: 1px solid #005a9e;
    border-radius: 4px;
    padding: 8px;
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #005a9e;
    border: 1px solid #003a6b;
}

/* 设置 QCheckBox 的字体和颜色 */
QCheckBox {
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #555555;
}

/* 设置 QComboBox 的背景色、边框和字体 */
QComboBox {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 5px;
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #333333;
}

/* 设置 QTableWidget 的背景色、边框和表头字体 */
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    font-family: Arial, sans-serif;
    font-size: 12px;
    color: #333333;
}

QHeaderView::section {
    background-color: #e0e0e0;
    border: 1px solid #cccccc;
    padding: 5px;
}

/* 设置 QVBoxLayout 和 QHBoxLayout 的间距 */
QVBoxLayout, QHBoxLayout {
    spacing: 10px;
}

/* 设置 QGridLayout 的间距 */
QGridLayout {
    spacing: 10px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QRadioButton" name="CM1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>模型替换</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>模型路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="path"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_load">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>是否勾选</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模块名称</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="CM2">
     <property name="text">
      <string>变量设置</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择变量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_var">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>替换数值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="v_var"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="CM3">
     <property name="text">
      <string>抓取号设置</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>吸嘴4抓取位</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>吸嘴1抓取位</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="le_4">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="cb_1">
       <property name="text">
        <string>启用使能</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>吸嘴3抓取位</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>吸嘴2抓取位</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="cb_3">
       <property name="text">
        <string>启用使能</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="cb_4">
       <property name="text">
        <string>启用使能</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="le_1">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="le_3">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="le_2">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="cb_2">
       <property name="text">
        <string>启用使能</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="CM4">
     <property name="text">
      <string>方案切换</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>方案选择</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="s_switch">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QRadioButton" name="CM5">
     <property name="text">
      <string>方案发送</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>方案选择</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="s_send">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
